modules/sugar/src/main/ts/ephox/sugar/api/selection/Awareness.ts

```diff
@@ -21,10 +21,12 @@ const isTextNodeWithCursorPosition = (el: SugarElement<Node>) => SugarText.getOp
   text.trim().length !== 0 || text.indexOf(Unicode.nbsp) > -1
 ).isSome();
 
+const isContentEditableFalse = (elem: SugarElement<Node>) => SugarNode.isHTMLElement(elem) && (elem.dom.contentEditable === 'false');
```

Comment:
Consider using `@ephox/sugar` API here.

```suggestion
const isContentEditableFalse = (elem: SugarElement<Node>) => SugarNode.isHTMLElement(elem) && (Attribute.get(elem, 'contenteditable') === 'false');
```